actions/bump-and-release-cloud-plugin/action.yaml

```diff
@@ -0,0 +1,320 @@
+
+    - name: Setup Node.js environment
+      uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+      with:
+        node-version: lts/iron
+        check-latest: true
+        cache: ${{ inputs.package-manager }}
+
+    - name: Setup Go environment
+      uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+      with:
+        go-version: ${{ inputs.go-version }}
+
+    - name: Install utilities
+      shell: bash
+      run: sudo apt-get install jq
+
+    - name: Is modern yarn
+      id: is-modern-yarn
+      shell: bash
+      if: inputs.package-manager == 'yarn'
+      run: |
+        version=$(yarn --version)
+
+        if [[ $version =~ ^1 ]]; then
+          echo "modern=false" >> $GITHUB_OUTPUT
+        else
+          echo "modern=true" >> $GITHUB_OUTPUT
+        fi
+
+    - name: Install dependencies
+      if: inputs.package-manager == 'yarn' && steps.is-modern-yarn.outputs.modern == 'true'
+      shell: bash
+      run: yarn install --immutable;
+
+    - name: Install dependencies
+      if: inputs.package-manager == 'yarn' && steps.is-modern-yarn.outputs.modern == 'false'
+      shell: bash
+      run: yarn install --frozen-lockfile;
+
+    - name: Install dependencies
+      if: inputs.package-manager == 'npm'
+      shell: bash
+      run: npm ci;
+
+    - name: Prep package.json
+      shell: bash
+      run: |
+        echo "$(jq 'del(.stableVersion)' package.json)" > package.json
+
+    - name: Bump version (old yarn)
+      if: inputs.package-manager == 'yarn' && steps.is-modern-yarn.outputs.modern == 'false'
+      shell: bash
+      run: |
+        yarn version --no-git-tag-version --${{ inputs.release-level }}
+
+    - name: Bump version
+      id: bump-version
+      if: inputs.package-manager != 'yarn' || steps.is-modern-yarn.outputs.modern == 'true'
+      shell: bash
+      run: |
+        ${{ inputs.package-manager }} version ${{ inputs.release-level }}
+
+    - name: Output version
+      id: output-version
+      shell: bash
+      run: |
+        export BUMPED_VERSION=$(cat package.json | jq -r '.version')
+        echo "version=${BUMPED_VERSION}" >> $GITHUB_OUTPUT
+        echo "version-tag=v${BUMPED_VERSION}" >> $GITHUB_OUTPUT
+
+    - name: Update changelog
+      if: inputs.update-changelog == 'true'
+      shell: bash
+      run: |
+        # Update the CHANGELOG to include the version we just created
+        sed -i "s/^## Unreleased\s*$/## Unreleased\n\n## ${{ steps.output-version.outputs.version-tag }}/" CHANGELOG.md
+
+    - name: Stage files
+      shell: bash
+      run: |
+        # We want to stage files now so we don't have to worry about generated files later
+        git add .
+
+    - name: Lint frontend
+      shell: bash
+      run: ${{ inputs.package-manager }} run lint
+
+    - name: Test frontend
+      shell: bash
+      run: ${{ inputs.package-manager }} run test:ci
```

Comment:
What about including optionally e2e tests execution?. There are some plugins that are already running playwright tests and other planning to use the new @grafana/plugin-e2e tool. Maybe include another input parameter?